src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+import java.util.Scanner;
+
 public class Duke {
+    /** Array of assignments that is limited to 100 tasks */
+    private static final Task[] ASSIGNMENTS = new Task[100];
+    /** Use to track the number of Task that is added */
+    private static int indexTask = 0;
+
+    /**
+     * Sorts the type of Task that is input by the user and prints out the respective types.
+     * Based on the keyword "todo", "deadline" or "event".
+     * Has helper functions addToDoTask(), addEventTask() and addDeadlineTask()
+     *
+     * @param userInput the string input by the user.
+     * @param splitUserInputs array of string that have been split into two.
+     */
+    public static void sortTypeOfTask(String userInput, String[] splitUserInputs) {
+        boolean isToDo = splitUserInputs[0].equals("todo");
+        boolean isDeadlines = splitUserInputs[0].equals("deadline");
+        boolean isEvent = splitUserInputs[0].equals("event");
+
+        System.out.println("\t Roger that. I've added this task:");
+
+        if(isToDo) {
+            addToDoTask(splitUserInputs);
+        } else if (isDeadlines) {
+            addDeadlineTask(splitUserInputs);
+        } else if (isEvent) {
+            addEventTask(splitUserInputs);
+        } else {
+            addNoTypeTask(userInput);
+        }
+        indexTask++;
+        System.out.println("\t Now you have " + indexTask + " in the list.");
+    }
+
+    /**
+     * Returns the indexTask that is .
+     *
+     * @param assignment assignment that is based on the type of task.
+     */
+    public static void addTask(Task assignment) {
+        ASSIGNMENTS[indexTask] = assignment;
+    }
+
+    /**
+     * Adds in the type of task which in this case no type task.
+     *
+     * @param userInput that is input by the user.
+     */
+    private static void addNoTypeTask(String userInput) {
+        addTask(new Task(userInput));
+        System.out.println("\t   [ ][ ] " + ASSIGNMENTS[indexTask].description);
+    }
+
+    /**
+     * Adds in the type of task which in this case Event task.
+     *
+     * @param splitUserInputs array of strings that is split into two.
+     */
+    private static void addEventTask(String[] splitUserInputs) {
+        addTask(new Event(splitUserInputs[1]));
+        ASSIGNMENTS[indexTask].markTypeTask();
+        System.out.println("\t   [" + ASSIGNMENTS[indexTask].getStatusOfTypeTask() + "]" + "[ ] "
+                + ASSIGNMENTS[indexTask].displayTypeTaskDetails());
```

Comment:
It becomes hard to understand what exactly is going on in this print statement because there are functions being executed inside the print statement itself. Maybe you could seperately execute the functions, store them in variables, and then print the variables? It would definitely make it easier to understand what is going on. :)